Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Operator/API/Driver.yaml

```diff
@@ -70,6 +70,7 @@ types:
   RespondHubRequest:
     status: RequestStatus
     driverId: Text
+    registrationNo: Text
```

Comment:
here `registrationNo` added because driver can have docs for many vehicles
After offline inspection, if docs verified, operator can activate vehicles